src/mv/mv.rs

```diff
@@ -360,6 +366,18 @@ fn rename(from: &PathBuf, to: &PathBuf, b: &Behaviour) -> Result<()> {
         }
     }
 
+    // "to" may no longer exist if it was backed up
+    if to.exists() && to.is_dir() {
+        // normalize behavior between *nix and windows
+        if from.is_dir() {
+            if is_empty_dir(to) {
+                try!(fs::remove_dir(to))
```

Comment:
Instead of adding another `try!` here, I'd prefer if you just change all the `try!`s to `?`.

Reply:
Ok, I'll look at refactoring that idiom.

Reply:
Changed as requested.
Tests all pass locally, CI pending.